web_src/css/admin.css

```diff
@@ -4,30 +4,6 @@
   margin: 12px -1rem -1rem;
 }
 
-.admin .table.segment {
-  padding: 0;
-  font-size: 13px;
-  overflow-x: auto;
```

Comment:
`overflow-x` is needed:

<img width="315" alt="Screen Shot 2023-04-27 at 09 01 24" src="https://user-images.githubusercontent.com/17645053/234734382-39319282-3869-4a59-8a8e-f2dae1147ed7.png">
<img width="376" alt="Screen Shot 2023-04-27 at 09 09 54" src="https://user-images.githubusercontent.com/17645053/234734389-52be9d90-7344-4f32-a311-ebfe5b4c5ff1.png">

Reply:
Good catch, luckily we only need one `overflow-x` now. Maybe we need to write a comment about how to test such style (I just forgot ...)

Reply:
Fix admin table layout #24370